features/bookmarks/bookmark_browse.feature

```diff
@@ -16,3 +15,8 @@ Feature: Browse Bookmarks
     When I go to the bookmarks page for user "ethel" with pseud "aka"
     Then I should see "Bookmarked with Other Pseud"
       And I should not see "Bookmarked with Default Pseud"
+
+  Scenario: When logged out, the "save" button on bookmarks does not show
```

Comment:
Please create a bookmark in the tag beforehand - otherwise there will never be a "Save" link.